src/scenes/bad-request-page/index.js

```diff
@@ -1,36 +1,59 @@
 import React, { Component } from 'react';
 import { Segment, Grid, Header, Button } from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledBadRequest = styled.div`
+  color: #00AA8D;
+
```

Comment:
Instead of using raw hex colors in code, try creating a `colors.js` file and add colors used in the entire site to it, and then import them in components normally.

export const topNavTeal = '#00AA8D'; // something like 

It will make maintaining the codebase easy (if you had to change a color, you just have to do it at one place).